site/src/pages/api/tmpdir.api.ts

```diff
@@ -0,0 +1,9 @@
+import { NextApiHandler } from "next";
+import os from "node:os";
+
+const handler: NextApiHandler = async (req, res) => {
+  res.write(os.tmpdir());
+  res.end();
+};
+
+export default handler;
```

Comment:
TODO: remove

Reply:
Done in https://github.com/blockprotocol/blockprotocol/pull/565/commits/a7c89be3da422059f4f2aaa25b44b038d6004ae6